lib/ember-addon-main.js

```diff
@@ -56,6 +56,11 @@ module.exports = {
     return this._cachedShouldColocateTemplates;
   },
 
+  getTemplateCompiler(inputTree, htmlbarsOptions) {
```

Comment:
If this is new public API we may want to:

* add details to the readme
* test that this "hook" can be used as expected

Reply:
* I've added details to the README.
* I didn't add a test case because I'm not sure how. There's no tests for that unit of code other than the one that runs the addon on the dummy application.